setup.py

```diff
@@ -25,13 +25,11 @@
 Set the aeneas package up.
 """
 
-from setuptools import Extension
-from setuptools import setup
-import io
 import os
 import shutil
-import sys
 
+from setuptools import setup, Extension
```

Comment:
This ``from .. import``, since it is from a standard lib, should be:

1. on top of the other imports, before imports that start with ``import``
2. one line per object imported